tests/test_memory.py

```diff
@@ -97,28 +97,6 @@ def render_foo(self, data):
         assert [r['foo'] for r in table.rows] == [-1, -2]
 
 
-def test_caches():
```

Comment:
Confirmed that we don't call .update():

```shell
$ git grep ".update()"
misc/bulkops.py:        self.trigger_update()
misc/bulkops.py:            self.trigger_update()
misc/counter.py:        '''Like dict.update() but add counts instead of replacing them.
pstat_deploy/ami.py:    ami.update()
pstat_deploy/aws.py:    healthcheck.update()
pstat_deploy/deployers/ec2.py:            seed_snapshot.update()
pstat_deploy/deployers/ec2.py:                seed_snapshot.update()
pstat_deploy/deployers/ec2.py:        instance.update()
selenium_tests/implementation/test_implementation.py:        def wait_for_button_update():
templates/document_control/document_admin_override.haml:                wym.update();
templates/document_control/document_edit.haml:                wym.update();
```

:+1: